style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: gray;
+}
```

Comment:
- [ ] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add a class to the heading and use that class in the .css file to style the heading.